templates/tensorflow-gpu-2.0/src/__ALGO__.py

```diff
@@ -0,0 +1,62 @@
+import tensorflow.keras.backend as K
```

Comment:
I tested this algorithm template in the nightly build on CPU mode, and it worked!